packages/wallet-ui/src/components/ui/organism/Header/SendModal/SendModal.view.tsx

```diff
@@ -116,6 +123,18 @@ export const SendModalView = ({ closeModal }: Props) => {
               decimalsMax={wallet.erc20TokenBalanceSelected.decimals}
               asset={wallet.erc20TokenBalanceSelected}
             />
+            <SeparatorSmall />
+            <div>
+              <label htmlFor="feeToken">Fee Token</label>
```

Comment:
Not a UI expert, but fee token seem not easy to understand as a user?

Reply:
Agree, I am proposing "Select Token for Transaction Fees"